app/models/user.rb

```diff
@@ -338,7 +314,7 @@ class User < ApplicationRecord
       ),
     )
   }
-  scope :with_feed, -> { where.not(feed_url: [nil, ""]) }
+  # scope :with_feed, -> { where.not(feed_url: [nil, ""]) }
```

Comment:
Should we delete this line instead of commenting it out? We can always get it back since it's in git.

```suggestion
```